autosubmit/log/log.py

```diff
@@ -271,7 +275,12 @@ def set_file(file_path: str, type='out', level="WARNING") -> None:
                     status_file_handler.addFilter(custom_filter)
                     Log.log.addHandler(status_file_handler)
                 os.chmod(file_path, 509)
-            except Exception as e:  # retry again
+            except Exception:  # retry again
+                retries += 1
+                if retries >= max_retries:
+                    raise AutosubmitCritical(
+                        message=f"Could not create log file {file_path} after {max_retries} attempts"
```

Comment:
Maybe we could add the exception to the message here at least, `message=f"Could not.... attempts: {str(e)}"?